kore/src/Kore/Attribute/SymbolData.hs

```diff
@@ -0,0 +1,124 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+{- |
+Description : Symbol declaration attributes
+Copyright   : (c) Runtime Verification, 2018
+License     : NCSA
+
+This module is intended to be imported qualified:
+@
+import qualified Kore.Attribute.SymbolData as Attribute
+@
+
+ -}
+
+module Kore.Attribute.SymbolData
```

Comment:
Why did this move?

Reply:
Circular dependencies: Attribute.Symbol imports MetadataTools which now needs the Symbol type, so it currently imports SymbolData. I could have written a .hs-boot file, but this seemed cleaner.

Reply:
That makes sense. The convention used for the other attributes modules would have this named `Kore.Attribute.Symbol.Symbol`, so that the module name still matches the type name.

Reply:
Done.